runtime/src/main/java/org/corfudb/runtime/LogReplicationLogicalGroupClient.java

```diff
@@ -238,10 +252,66 @@ public void addDestination(String logicalGroup, List<String> remoteDestinations)
     }
 
     /**
-     * Remove a destination from the LogReplicationModelMetadataTable for a specific group.
+     * Sets the destinations for a given logical group to the provided list.
+     * <p>
+     * A new logical group will be created if one does not already exist.
+     * Setting destinations for a logical group will overwrite existing destinations.
+     * Can be used to clear existing destinations by passing in an empty list.
+     * Clearing existing destinations will remove the logical group.
+     *
+     * @param logicalGroup The group of tables to be replicated, represented as a string.
```

Comment:
isn't this more like: logical tag for group of tables to be replicated?... otherwise it sounds like it represents the set/group of tables to replicate